pytorch_lightning/trainer/trainer.py

```diff
@@ -908,6 +911,9 @@ def test(self, model=None):
         """
         self.testing = True
         if model is not None:
```

Comment:
there is no need for this if/else split, just call `eval_results = self.run_evaluation(test=True, model=model)` and if the model is none that it does the same as default parameter...

Reply:
Thanks. I will fix it.